.github/CODEOWNERS

```diff
@@ -289,6 +289,7 @@ yarn.lock                                                @getsentry/owners-js-de
 /static/app/views/replays/        @getsentry/replay-frontend
 /src/sentry/replays/              @getsentry/replay-backend
 /tests/sentry/replays/            @getsentry/replay-backend
+/src/sentry/userfeedback/         @getsentry/replay-backend
```

Comment:
Let's standardize on the `user_feedback` naming convention.

Reply:
@JoshFerge in agreement with changing from `userfeedback` to `user_feedback`?

Reply:
yep makes sense.